src/shared/ListRepo/ReposTableTeam/ReposTableTeam.tsx

```diff
@@ -0,0 +1,286 @@
+import {
+  createColumnHelper,
+  flexRender,
+  getCoreRowModel,
+  getSortedRowModel,
+  useReactTable,
+} from '@tanstack/react-table'
+import type { SortingState } from '@tanstack/react-table'
+import cs from 'classnames'
+import isEmpty from 'lodash/isEmpty'
+import PropTypes from 'prop-types'
+import { useContext, useMemo, useState } from 'react'
+import { useParams } from 'react-router-dom'
+
+import {
+  OrderingDirection,
+  Repository,
+  TeamOrdering,
+  useReposTeam,
+} from 'services/repos'
+import { ActiveContext } from 'shared/context'
+import { formatTimeToNow } from 'shared/utils/dates'
+import Button from 'ui/Button'
+import Icon from 'ui/Icon'
+
+import InactiveRepo from '../InactiveRepo'
+import { repoDisplayOptions } from '../ListRepo'
+import NoReposBlock from '../NoReposBlock'
+import 'ui/Table/Table.css'
+import RepoTitleLink from '../RepoTitleLink'
+
+export function getSortingOption(
+  sorting: Array<{ id: string; desc: boolean }>
+) {
+  const state = sorting.at(0)
+
+  if (state) {
+    const direction = state?.desc
```

Comment:
Do you need `state?` since you are already checking if state is defined. All good if the IDE is showing an error though.